tests/fixtures/sample-restrictions.json

```diff
@@ -0,0 +1 @@
+[{"name":"mock_post_order","version":1,"rawRestrictions":[{"SenderOwnsAllInputs":[]},{"SenderHasInputRole":[{"index":0,"roleKey":"Supplier"}]},{"op":"and"},{"FixedOutputMetadataValueType":[{"index":0,"metadataKey":"SomeMetadataKey","metadataValueType":"Literal"},{"index":0,"metadataKey":"SomeOtherMetadataKey","metadataValueType":"File"}]},{"op":"and"},{"op":"and"}]},{"name":"mock_accept_order","version":1,"rawRestrictions":[{"SenderOwnsAllInputs":[]},{"op":"or"},{"SenderHasInputRole":[{"index":0,"roleKey":"Supplier"}]}]}]
```

Comment:
not used anywhere? Though it's nice to have a pure JSON example of a valid program since that's what the user inputs, maybe in the README?

Reply:
Yea, the intention was to parse file as an arg. Will include in the read me

Reply:
this doesn't seem resolved? It's not in the readme and the format of the json is the old style